### DISCUSSION TITLE
CLI: mcap info should be clearer about what kind of mcap it is

### DISCUSSION THREAD

**Discussion by wkalt** [foxglove/mcap#1591] [General]
The `mcap info` command tries to get as much useful info as it can out of the summary section of the file. Depending on various recording choices (chunking, compression, indexing) some info may be there and some may not be. When info is not available it's just omitted from the info output, so that running `mcap info` on an indexed file gives you a richer output than on an unindexed one.

It would be nice if we could add to the output some kind of enum that describes the variety of file, for instance

* unchunked
* chunked
* chunked+indexed

if those are in fact exhaustive. That would make it easy to see at a glance that a file is unchunked, vs something odd going on with the tool.

**Comment by jhurliman** [comment]
Summarizing a Slack discussion:

- If a file is unindexed, simply report it as unindexed
- If a file has an index, look for chunk index records (or the absence thereof) to report it as either chunked or unchunked
